lib/puppet/util/windows/api_types.rb

```diff
@@ -196,6 +196,9 @@ def self.from_wide_string(wstr)
   FFI.typedef :uchar, :byte
   FFI.typedef :uint16, :wchar
 
+  # https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/32862b84-f6e6-40f9-85ca-c4faf985b822
+  FFI.typedef :ulong, :dword
```

Comment:
This is already defined in https://github.com/puppetlabs/puppet/blob/1215b8bcb2ad02a66296ffbe7019a5734bf4ab43/lib/puppet/util/windows/api_types.rb#L145